instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -300,28 +300,42 @@ def get_connection_attributes(self, connection):
             self.span_attributes[SpanAttributes.NET_PEER_PORT] = port
 
 
+class _TracedConnectionProxy:
+    pass
+
+
 def get_traced_connection_proxy(
     connection, db_api_integration, *args, **kwargs
 ):
     # pylint: disable=abstract-method
-    class TracedConnectionProxy(wrapt.ObjectProxy):
-        # pylint: disable=unused-argument
-        def __init__(self, connection, *args, **kwargs):
-            wrapt.ObjectProxy.__init__(self, connection)
+    class TracedConnectionProxy(type(connection), _TracedConnectionProxy):
+        def __init__(self, connection):
+            self._connection = connection
 
-        def cursor(self, *args, **kwargs):
-            return get_traced_cursor_proxy(
-                self.__wrapped__.cursor(*args, **kwargs), db_api_integration
+        def __getattr__(self, name):
+            return object.__getattribute__(
+                object.__getattribute__(self, "_connection"), name
             )
 
         def __enter__(self):
-            self.__wrapped__.__enter__()
+            self._connection.__enter__()
             return self
 
-        def __exit__(self, *args, **kwargs):
-            self.__wrapped__.__exit__(*args, **kwargs)
+        def __exit__(self, exc_type, exc_value, traceback):
+            self._connection.__exit__(exc_type, exc_value, traceback)
```

Comment:
If you already have defined `def __getattr__(self, name):`, why is this needed without additional logic?

Reply:
Good point, removing.